### DISCUSSION TITLE
Help In 2FA

### DISCUSSION THREAD

**Discussion by Ghost00Shell** [community/community#78145] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

**My country (EGYPT) is not listed in 2FA HELP!!**

**Comment by DNin01** [comment]
It looks like you're having trouble setting up 2FA via SMS because GitHub isn't able to reliably deliver text messages to your country. Try configuring an authenticator (TOTP) app during 2FA setup instead. There are various TOTP app options available for both mobile devices and computers.

**Comment by Ghost00Shell** [comment]
same problem

**Comment by Ghost00Shell** [comment]
how i can contact github support team

**Reply by Akidul-islam** [reply]
GitHub primarily provides support through their help resources and community forums. You can visit the GitHub Support page at https://support.github.com/ to find documentation, guides, and ways to contact support if necessary. Keep in mind that GitHub doesn't offer direct email or phone support for free accounts. If you have a paid plan, you may have access to additional support options.

**Comment by queenofcorgis** [comment, marked as answer]
If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).
